actor/src/lib.rs

```diff
@@ -1,6 +1,5 @@
-//!
 //! Iroha simple actor framework.
-//!
+#![allow(clippy::same_name_method)]
```

Comment:
you can remove this

Reply:
I'll keep it for the time being, if you ran `cargo clippy -- -W clippy::same_name_method`, it would exclude this file, but keep highlighting others.